.changeset/perfect-snails-act.md

```diff
@@ -0,0 +1,5 @@
+---
+'@crowdstrike/glide-core': minor
+---
+
+Inline Alert no longer supports the `removable` attribute. It will no longer dispatch a `remove` event either.
```

Comment:
> It will no longer dispatch a `remove` event either.

Should be implied, but better to be explicit when we remove things so that event handlers can be cleaned up, I think.